ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -748,6 +748,11 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent) {
                   new ReactTextInputSubmitEditingEvent(
                       editText.getId(),
                       editText.getText().toString()));
+              // if blurOnSubmit is false, and isMultiline, go to new line
```

Comment:
ah, I think I found what you mean..
> !blurOnSubmit && isMultiline => return false; // Perform default behaviour.
> !blurOnSubmit && !isMultiline => return true; // Shallow <Enter>.

should not generate submit event, right? I will fix them asap